src/test/java/com/xceptance/neodymium/util/SelenideAddonsTest.java

```diff
@@ -640,4 +642,33 @@ public void testOpenHtmlContentWithCurrentWebDriver()
         SelenideAddons.openHtmlContentWithCurrentWebDriver(textHtml);
         Assert.assertEquals(text, $("body").getText());
     }
+
+    @Test
+    public void testOptionalWaitUntil()
+    {
+        openBlogPage();
+        SelenideElement privaceDialog = $("#privacy-message");
+        boolean isVisible = SelenideAddons.optionalWaitUntilCondition(privaceDialog, visible);
+        assertTrue(isVisible);
+        long startTime = new Date().getTime();
+        boolean isHidden = SelenideAddons.optionalWaitUntilCondition(privaceDialog, hidden);
+        long endTime = new Date().getTime();
+        assertFalse(isHidden);
+        Assert.assertTrue(endTime - startTime > Neodymium.configuration().optionalElementRetryTimeout());
```

Comment:
use static import like you did in `assertTrue(isVisible);`

Reply:
done